Backend/Remora.Discord.API.Abstractions/API/Objects/Users/IUser.cs

```diff
@@ -39,7 +39,7 @@ public interface IUser : IPartialUser
     new Snowflake ID { get; }
 
     /// <summary>
-    /// Gets the username of the user. This is not a unique value.
+    /// Gets the username of the user. This is a unique value.
```

Comment:
It's not a unique value *yet* - we should avoid this wording until the migration is complete.

Reply:
Git appears to have eaten my code
![image](https://github.com/Remora/Remora.Discord/assets/42438262/b8121e3b-0396-472a-ba95-d1cee01d2c8c)